ATP.BusinessLogicLayer/Services/ManagerService.cs

```diff
@@ -0,0 +1,118 @@
+﻿using ATP.DataAccessLayer.Enum;
+using ATP.DataAccessLayer.Models;
+
+namespace ATP.BusinessLogicLayer.Services;
+
+public class ManagerService
+{
+    private readonly List<Flight> availableFlights;
+    public ManagerService(List<Flight> availableFlights, BookingService bookingService)
+    {
+        this.availableFlights = availableFlights;
+    }
+
+    public void RunMenu()
+    {
+        while (true)
+        {
+            Console.WriteLine(@"Manager Menu
+--------------------------------------
+1. Filter by Flight
+2. Filter by Price
+3. Filter by Class
+4. Exit");
+            Console.Write("Enter your choice: ");
+
+            string choice = Console.ReadLine();
+            switch (choice)
+            {
+                case "1":
+                    FilterByFlight();
+                    break;
+                case "2":
+                    FilterByPrice();
+                    break;
+                case "3":
+                    FilterByClass();
+                    break;
+                case "4":
+                    Console.WriteLine("Exiting Manager Menu.");
+                    return;
+                default:
+                    Console.WriteLine("Invalid choice. Please try again.");
+                    break;
+            }
+        }
+    }
+
+    private void FilterByFlight()
+    {
+        Console.WriteLine("Filter by Flight:");
+        Console.Write("Enter Departure Country: ");
+        string departureCountry = Console.ReadLine();
+
+        Console.Write("Enter Destination Country: ");
+        string destinationCountry = Console.ReadLine();
+
+        var filteredFlights = availableFlights.FindAll(flight =>
+            flight.DepartureCountry.Equals(departureCountry, StringComparison.OrdinalIgnoreCase) &&
+            flight.DestinationCountry.Equals(destinationCountry, StringComparison.OrdinalIgnoreCase)
+        );
+
+        DisplayFilteredFlights(filteredFlights);
+    }
+
+    private void FilterByPrice()
+    {
+        Console.WriteLine("Filter by Price:");
+        Console.Write("Enter Minimum Price: ");
+        if (!double.TryParse(Console.ReadLine(), out double minPrice))
+        {
+            Console.WriteLine("Invalid input for minimum price.");
+            return;
+        }
+
+        Console.Write("Enter Maximum Price: ");
+        if (!double.TryParse(Console.ReadLine(), out double maxPrice))
+        {
+            Console.WriteLine("Invalid input for maximum price.");
+            return;
+        }
+
```

Comment:
Consider adding a check to ensure that `minPrice` is not greater than `maxPrice`.